MCInst.c

```diff
@@ -165,6 +165,12 @@ void MCOperand_setFPImm(MCOperand *op, double Val)
 	op->FPImmVal = Val;
 }
 
+MCExpr *MCOperand_getExpr(const MCOperand *MC)
+{
+	assert(0 && "unimplemented expr");
+	return NULL;
```

Comment:
It turned out that `SystemZ` and some `AArch64` instructions (I think) now also use the expressions. But emit the immediate value directly in LLVM, if symbol information is missing. With the SystemZ PR is will be:
```c
int64_t MCOperand_getExpr(const MCOperand *op)
{
	return op->ImmVal;
}
```